app/assets/javascripts/proxy_status.js

```diff
@@ -0,0 +1,103 @@
+function setItemStatus(item, response) {
+  if(response.success) {
+    item.attr('title', __('Active'));
+    item.addClass('text-success');
+    item.html(icon('ok-sign'));
+  } else {
+    item.attr('title', response.message);
+    item.addClass('text-danger');
+    item.html(icon('exclamation-sign'));
+  }
+  item.tooltip({html: true});
+}
+
+function setProxyVersion(item, response) {
+  item.attr('title',response.message.version);
```

Comment:
why set the version both here and in the text? I find it confusing to see the same value on hover.